components/api/src/main/java/com/hotels/styx/api/ContentOverflowException.java

```diff
@@ -16,7 +16,12 @@
 package com.hotels.styx.api;
 
 /**
- * An exception that indicates that a content-aggregation operation exceeded the maximum permitted amount of content.
+ * Thrown when HTTP content aggregation fails because received more data than excepted.
+ * <p>
+ * A streaming HTTP message can  be aggregated and converted (aggregated) to a
```

Comment:
1. Double space between "can" and "be".
2. The word "aggregated" appears twice.